lib/ansible/modules/windows/win_service.py

```diff
@@ -59,6 +59,14 @@
       - restarted
     default: null
     aliases: []
+  delayed:
+    description:
+      - If True, when start_mode is C(auto) the service is flagged for a delayed start.
+    default: false
+    required: false
+    choices: [ 'True', 'False' ]
```

Comment:
Good question. That's one of the things we have to decide, I guess. And when decided we will modify all other modules to comply with this for consistency IMO.